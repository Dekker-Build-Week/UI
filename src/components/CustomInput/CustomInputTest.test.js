import React from 'react';

import CustomInput from './CustomInput';
import renderer from 'react-test-renderer';

describe('Custom Inputs test suite', () => {
    it('Renders the CustomInput correctly', () => {
        const tree = renderer
            .create(<CustomInput> </CustomInput>)
            .toJSON();

        expect(tree).toMatchSnapshot();
    })
});