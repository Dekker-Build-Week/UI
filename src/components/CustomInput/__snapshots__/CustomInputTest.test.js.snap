// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Custom Inputs test suite Renders the CustomInput correctly 1`] = `
<div
  className="MuiFormControl-root makeStyles-formControl-10"
>
  <div
    className="MuiInputBase-root MuiInput-root makeStyles-marginTop-9 MuiInput-underline makeStyles-underline-2 MuiInputBase-formControl MuiInput-formControl"
    onClick={[Function]}
  >
    <input
      aria-invalid={false}
      className="MuiInputBase-input MuiInput-input"
      disabled={false}
      onBlur={[Function]}
      onChange={[Function]}
      onFocus={[Function]}
      required={false}
      type="text"
    />
  </div>
</div>
`;
